pkgs/by-name/yt/ytui-music/package.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, sqlite
+, stdenv
+, darwin
+, mpv
+, youtube-dl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ytui-music-unstable";
```

Comment:
```suggestion
  pname = "ytui-music";
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/256702/commits/66733f9113ecfd9f63910008c2056bd3767d3676